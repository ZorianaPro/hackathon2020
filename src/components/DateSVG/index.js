import React from 'react';

const DateSVG = (props) => (
  <svg className="DateSVG"
    {...props}>
    <use xlinkHref="#DateSymbol" />
  </svg>
);

export default DateSVG;
