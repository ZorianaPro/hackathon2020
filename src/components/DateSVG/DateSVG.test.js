import React from 'react';
import { shallow } from 'enzyme';
import DateSVG from './';

describe('DateSVG', () => {
  it('renders without crashing', () => {
    expect(shallow.bind(shallow, <DateSVG />)).not.toThrow();
  });

  it('renders as expected', () => {
    expect(shallow(<DateSVG />)).toMatchSnapshot();
  });
});
