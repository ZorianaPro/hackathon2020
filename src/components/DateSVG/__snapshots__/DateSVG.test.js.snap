// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DateSVG renders as expected 1`] = `
<svg
  className="DateSVG"
>
  <use
    xlinkHref="#DateSymbol"
  />
</svg>
`;
